### PR TITLE
LPS 131505 and LPS-130739

### PR DESCRIPTION
**Pull request by brunofariasdeo**
Related Issues: 
https://issues.liferay.com/browse/LPS-131505
https://issues.liferay.com/browse/LPS-130739

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java [hunk 1]
```diff
@@ -230,6 +234,7 @@ protected void initModels() {
 		setDDMFormValuesDefaultLocale();
 		setDDMFormValuesAvailableLocales();
 
+		setDDMFormInstanceId();
 		setGroupId();
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java:238]: @brunofariasdeo, are we able to remove the the blank lines of the method `initModels` and also sort the methods invocations alphabetically?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/page/change/DDMFormPageChangeTracker.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+package com.liferay.dynamic.data.mapping.form.page.change;
+
+/**
+ * @author Bruno Oliveira
+ * @author Carolina Barbosa
+ */
+public interface DDMFormPageChangeTracker {
+
+	public DDMFormPageChange getDDMFormPageChangeById(String ddmFormInstanceId);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/page/change/DDMFormPageChangeTracker.java:23]: @brunofariasdeo, please rename the method to `getDDMFormPageChangeByDDMFormInstanceId` to keep consistency with the parameter renaming.

**Review by natocesarrego — CHANGES_REQUESTED** [review]
@brunofariasdeo, please see my comments and check out the test results. I think that one of them is related to this pull.

Thanks.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java (hunk 1 above)
Comment (brunofariasdeo) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java:238]: Eagle eyes! I've changed it here :)

**Pull request closed without merging**